src/Model/Contact.php

```diff
@@ -453,6 +453,13 @@ public static function getBasepath($url, $dont_update = false)
 	 */
 	public static function isLocal($url)
 	{
+		if (!parse_url($url, PHP_URL_SCHEME)) {
+			$addr_parts = explode('@', $url);
+			if (count($addr_parts) == 2) {
+				return $addr_parts[1] == DI::baseUrl()->getHostname();
+			}
```

Comment:
This should return no matter what because we don't have a scheme so the `compareLink` will fail anyway.
```suggestion
			return count($addr_parts) == 2 && $addr_parts[1] == DI::baseUrl()->getHostname();
```

Reply:
Done